### PR TITLE
Updating GC logging options for detailed logging

### PR DESCRIPTION
**Pull request by nitinsingh130519**

### CODE REVIEW

**Comment by nitinsingh130519** [comment]
@kthatipally

File: perf/renaissance-jmh/gen_playlist.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
     file.write("\t\t<testCaseName>renaissance-jmh-" + bench + "</testCaseName>\n")
     file.write("\t\t<command>\n")
-    file.write("\t\t\t$(JAVA_COMMAND) $(ADD_OPENS_CMD) $(JVM_OPTIONS) -Xlog:gc*:file=" + bench + "_gc.log -XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages")
+    file.write("\t\t\t$(JAVA_COMMAND) $(ADD_OPENS_CMD) $(JVM_OPTIONS) -Xlog:gc*,gc+ref=debug,gc+phases=debug,gc+age=trace,gc+ergo*=trace,safepoint:file=gc.log" + bench + "_gc.log -XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages")
```
Comment (dhanalla) [inline perf/renaissance-jmh/gen_playlist.py]: remove gc.log as bench+"_gc.log" already takes care of passing gc file specific to bench.
gc+ref=debug,gc+phases=debug,gc+age=trace,gc+ergo*=trace,safepoint:file=**gc.log**

**Review by dhanalla — CHANGES_REQUESTED** [review]

**Review by dhanalla — APPROVED** [review]

**Comment by karianna** [comment]
reminder to upstream this to Adoptium if appropriate

**Comment by gdams** [comment]
@nitinsingh130519 do you not need to run the python updater script to actually update the playlist.xml file?

**Pull request merged**